server/athenian/api/align/goals/measure.py

```diff
@@ -346,3 +364,12 @@ def _team_tree_prune_empty_branches(
         return team_tree.with_children(kept_children)
     else:
         return None
+
+
+def _parse_metric_params(metric_params: Optional[dict]) -> Optional[dict]:
+    if not metric_params:
+        return None
+    parsed = metric_params.copy()
+    if isinstance(threshold := metric_params.get("threshold"), str):
```

Comment:
Let's move this "threshold" somewhere as a constant string

Reply:
Otherwise, we've got an abstraction leak here: we parse input directly in the business logic of the goal metrics. Ideally, we would require the caller to do this beforehand.